### DISCUSSION TITLE
Is it possible to do massflash?

### DISCUSSION THREAD

**Discussion by CHNtentes** [OE4T/meta-tegra#1301] [Q&A]
Hi.

It seems that with Nvidia stock L4T, it's possible to use --massflash to flash multiple devices at the same time.

Is it possible to do that with meta-tegra?

Thanks in advance.

**Comment by dwalkes** [comment]
There was some discussion about this in the past at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2021-10-14 - I think Ridgerun had done some work on this but I don't recall seeing a pull request and can't find it with a quick search.

**Comment by madisongh** [comment]
The flashing scripts we have support the `--usb-instance` option, so in theory, at least, you should be able to create a script that identifies multiple jetsons, populates directories with the appropriate tegraflash package contents for each, and starts up per-USB-instance flashes in parallel.
